tutorials/script/core/10_multiple_fallback_nodes.py

```diff
@@ -0,0 +1,127 @@
+# %% [markdown]
+"""
+# Core: 10. Multiple fallback nodes
+
+This tutorial shows basics of creating a separate fallback node for every flow. In this case it is done via local transitions (LINK).
+
+Let's do all the necessary imports from DFF:
+"""
+
+# %pip install dff
+
+
+"""Importing all the necessary modules:"""
+
+# %%
+from dff.pipeline import Pipeline
+from dff.script import TRANSITIONS, RESPONSE, Message
+import dff.script.conditions as cnd
+
+# %% [markdown]
+"""
+Let's create a script that will consist of several flows, that can be accessed through dialogue:
+"""
+
+# %%
+toy_script = {
+    "greeting_flow": {
+        "start_node": {
+            RESPONSE: Message(),
+            TRANSITIONS: {
+                ("greeting_flow", "say_hi_node"): cnd.true()
+            }
+        },
+
+        "say_hi_node": {
+            RESPONSE: Message(text="Hello, how can I help you?"),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"): cnd.exact_match(Message(text="auth")),
+                ("information_flow", "response_node"): cnd.exact_match(Message(text="info")),
+                ("fallback_node"): cnd.true()
+            }
+        },
+
+        "fallback_node": {
+            RESPONSE: Message(text="Unknown Command. Try again."),
+            TRANSITIONS: {"say_hi_node": cnd.exact_match(Message(text="hi"))
+            }
+        }
+    },
+    "authorization_flow": {
+        "response_node": {
+            RESPONSE: Message(text="Write your name, please."),
+            TRANSITIONS: {
+                ("authorization_flow", "auth_success_node"):
+                    cnd.exact_match(Message(text="admin")),
+                ("authorization_fallback_node"):
+                    cnd.true()
+            }
+        },
+
+        "auth_success_node": {
+            RESPONSE: Message(text="Authentication successful."),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"):
+                    cnd.exact_match(Message(text=""))
+            }
+        },
+
+        "authorization_fallback_node": {
+            RESPONSE: Message(text="User not found."),
+            TRANSITIONS: {
+                ("authorization_flow", "response_node"):
+                    cnd.exact_match(Message(text=""))
+            }
+        }
+    },
+
+    "information_flow": {
+        "response_node": {
+            RESPONSE: Message(text="What information you would like to know?"),
+            TRANSITIONS: {
+                ("information_flow", "weather_node"):
+                    cnd.exact_match(Message(text="weather")),
+                ("information_flow", "time_node"):
+                    cnd.exact_match(Message(text="time")),
+                ("information_fallback_node"):
+                    cnd.true()
```

Comment:
Script should be rewritten in a way that shows how using `LOCAL` transitions simplifies script writing process.

The current script has exactly one transition to fallback node per flow so it is not clear how using `LOCAL` helps.
If the script had several nodes that have to have a fallback then we could showcase that using `LOCAL` saves time from writing the same transitions in every node as it is suggested in the next markdown cell.